io-utils/src/http.rs

```diff
@@ -145,13 +145,14 @@ where
 
         transfer
             .write_function(|buf| {
-                received_body.push_str(from_utf8(buf).expect({
-                    trace!(
+                received_body.push_str(from_utf8(buf).unwrap_or_else(|_| {
+                    panic!("{}", {
+                        trace!(
                         "Error converting data {:?} from UTF-8.  Continuing with default value.",
                         buf
                     );
-
-                    &format!("Error converting data {:?} from UTF-8.", buf)
+                        &format!("Error converting data {:?} from UTF-8.", buf)
```

Comment:
Is the indentation meaningful, or a mistake?  Have you run `cargo fmt` on the files?

Reply:
yes I ran `cargo fmt` . The indentation for `format!` is meaningful since it is placed under `panic!` in line 149. I think the wrong indentation is with `trace!`, right ?

Reply:
Ah OK, sorry, yeah I misread the diff and thought the bracket on line 153 was associated with `panic!` but it's actually associated with `trace!`.  If you've ran `cargo fmt` then we'll just leave it, and I'll mark this as resolved.